styles/draft style

```diff
@@ -0,0 +1,177 @@
+.wrapper {
+  position: absolute;
+  background-color: #fff;
+  background-image: url(images/Header.svg);
+  background-size: contain;
+  background-position: top;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: center;
+}
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - The font size of your project text and the size of the buttons are very small which does not match the Figma design.
  - The margin between the company (eg Canopy) and the role (Backend dev) does not match the Figma design.
    
  | Your Design |Figma Design |
  | :---:   | :---: | 
  | ![image](https://user-images.githubusercontent.com/92303891/219904671-74bd20b2-ba46-4c03-bc99-1009594f0adf.png) | ![image](https://user-images.githubusercontent.com/92303891/219904756-82e5999c-9340-40d8-9709-c1edff3387f9.png) |